### DISCUSSION TITLE
Wrong Permissions listed for creating new repository from template

### DISCUSSION THREAD

**Discussion by schlichtanders** [community/community#61685] [Apps, API and Webhooks]
### Select Topic Area

Bug

### Body

The REST API `POST /repos/{template_owner}/{template_repo}/generate` is [wrongly listed](https://docs.github.com/en/rest/overview/permissions-required-for-github-apps?apiVersion=2022-11-28#repository-permissions-for-metadata):

![image](https://github.com/community/community/assets/4255480/357e62fe-bb03-4392-97ed-aa7dcfe8f912)

- currently under `Repository Metadata` scope
- marked as `read-only`

Adding these permissions is not enough. I tried it and it failed with return code 403 "Resource not accessible by integration". I am currently trying `Repository Adminstration` `read-write` scope instead (EDIT: works actually). 

EDIT: ~Repository Adminstration scope does not seem to be enough... I am stuck.~ Actually it works with Repo Admin permissions. You need to make sure to throw away all old user access tokens before testing.

**Comment by moonfury-ops** [comment]
Thank you
